src/base/rss/rss_autodownloader.h

```diff
@@ -99,6 +99,8 @@ namespace RSS
         void store();
         void storeDeferred();
 
+        static QString generateSavePath(const QString savePath, const QVariantHash articleData);
```

Comment:
generateSavePath shouldn't be static member of AutoDownloader. Change it to free function in anonymous namespace inside "rss_autodownloader.cpp".